tests/envs/maze/test_point_maze.py

```diff
@@ -39,3 +39,23 @@ def test_goal_cell():
     obs = env.reset(options={"goal_cell": [2, 1]}, seed=42)[0]
     desired_goal = np.array([-0.36302198, -0.53056078])
     np.testing.assert_almost_equal(desired_goal, obs["desired_goal"], decimal=4)
+
+
+def test_multiprocessing_not_crash():
```

Comment:
The Problem with this kind of nondeterministic testing, is that I can verify it working or not 

I can not get it to fail with the version of maze

If it can not prove that a bug has been fixed, it should not be added

Reply:
let me see if I can make a test that fails it, I think the actual problem is that when using Tianshou RL it recreates the env a bunch of times, so what I should be doing to test is creating a buttload of envs.